drivers/Serial.c

```
@@ -37,8 +38,43 @@
 static uint16_t serial_port = COM1;
 static int serial_initialized = 0;
 
+static int SerialDevRead(struct CharDevice* dev, void* buffer, uint32_t size) {
```

Clang (14.0.6)

[warning] parameter 'dev' is unused

(misc-unused-parameters)